evi-typescript-example/package.json

```
@@ -9,7 +9,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "hume": "^0.7.2"
+    "hume": "^0.8.1"
```

```suggestion
    "hume": "^0.8.3"
```
The most recent version is now 0.8.3, according to [the TypeScript SDK repo](https://github.com/HumeAI/hume-typescript-sdk)

---

I tested it on my end, and it works with version 0.8.3. It would be important to regenerate the pnpm-lock.yaml file as well to correspond with it, if the decision is made to commit this change